chunk/tag.go

```diff
@@ -113,7 +115,7 @@ func (t *Tag) Total() int64 {
 func (t *Tag) DoneSplit(address Address) int64 {
 	total := atomic.LoadInt64(&t.split)
 	atomic.StoreInt64(&t.total, total)
-	t.Address = address
+	t.Address = hex.EncodeToString(address)
```

Comment:
ditto

Reply:
Misunderstood the comment. changing it back to Address.